pkg/defaultcomponents/defaults.go

```diff
@@ -72,6 +73,22 @@ import (
 	"go.uber.org/multierr"
 )
 
+var datadogExporterFeatureGateDeprecation = featuregate.GlobalRegistry().MustRegister("adot.exporter.datadogexporter.deprecation",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("Allows for the ADOT Collector to be configured and started with the Datadog Exporter"))
```

Comment:
```suggestion
	featuregate.WithRegisterDescription("Removes the Datadog exporter from the set of configurable exporters"))
```

This should be written describing what happens when the gate is enabled.  The same change applies to all of these descriptions.